src/app/workflow/version-modal/version-modal.component.ts

```diff
@@ -124,4 +124,13 @@ export class VersionModalComponent implements OnInit, AfterViewChecked {
     }
   }
   // Validation ends here
+
+  // Checks if the currently edited test parameter file already exists
+  hasDuplicateTestJson() {
```

Comment:
Isn't this just `return this.testParameterFilePaths.includes(this.testParameterFilePath)`?

Reply:
Some weird merge caused this to appear.  I've rebased to remove irrelevant code